examples/webgpu_compute_texture.html

```diff
@@ -95,7 +95,7 @@
 				const plane = new THREE.Mesh( new THREE.PlaneGeometry( 1, 1 ), material );
 				scene.add( plane );
 
-				renderer = new WebGPURenderer( { antialias: true } );
+				renderer = new WebGPURenderer( { antialias: true, preferWebGL: true } );
```

Comment:
I imagine you are debugging perhaps with `forceWebGL` instead of `preferWebGL`?

Reply:
Oops my bad my local as a lot of `WIP` and I pushed the wrong file in my previous commit. 😅